src/shared/services/WalletService/WalletService.ts

```diff
@@ -19,10 +19,11 @@ function createWalletService(mirrorNodeService: MirrorNodeServiceType) {
     return hashconnect.getSigner(AccountId.fromString(accountId));
   };
 
-  const reconnectToOtherNetwork = async (network: LedgerId) => {
-    await disconnect();
-    localStorage.setItem("activeNetwork", network?.toString());
-    setTimeout(() => window.location.reload(), 1000);
+  const reconnectToOtherNetwork = (network: LedgerId) => {
```

Comment:
I've noticed that `DexService` has this stub function, is it needed at all? As I don't see any other parts of Wallet service added there.

```
    // eslint-disable-next-line @typescript-eslint/no-unused-vars
    reconnectToOtherNetwork: async (_network: LedgerId) => {},
```